.size-limit.json

```diff
@@ -9,7 +9,7 @@
     "name": "> all css",
     "path": ".next/static/**/*.css",
     "running": false,
-    "limit": "41 kB"
+    "limit": "45 kB"
```

Comment:
probably to update something else in size-limit config, CI still failing

Reply:
weird:
https://github.com/babajka/babajka-frontend/pull/124/checks?check_run_id=1584722781#step:3:1684